### ISSUE TITLE
Combine gaze information and the features extracted from backbone

### ISSUE THREAD

**Issue by shivahanifi** [shivahanifi/GazeMDETR#2]
Consider a simple ratio relation between the image patches and the feature map. multiply the gaze heatmap and get the results.

- The attention heatmap needs to be downsampled to the dimension of the feature map, also flattened, and then multiply

**Comment by shivahanifi** [comment]
# Approach:

|parameter|value|Notes|
|---|---|---|
|gaze|torch.Size([3, 480, 640])|normalized `norm_map` using `transforms.ToTensor`|
|src|torch.Size([850, 1, 256])||

**Comment by shivahanifi** [comment]
Pixe/patch <-----> Attention heatmap
Pixe/patch <-----> feature map

Attention heatmap <-----> feature map

**Comment by shivahanifi** [comment]
Using the `norm_map_gray` and transforming it as:
```
transform_normMap = T.Compose([
    T.Resize(800),
    T.ToTensor(),
    T.Normalize((-1), (2))
])
```
and then passing it as the `gaze` parameter inside the `plot_inference` function, and  `mdetr.py` and `transformer.py` modules. 

- `transformer.py` module

    ```
    gaze = torch.nn.functional.interpolate(gaze.unsqueeze(0),size=(h,w), mode='bilinear', align_corners=False).squeeze(0)
    gaze = gaze.unsqueeze(0).to(device)
    src= gaze * src
```
